test/sanity/run-test.sh

```diff
@@ -47,4 +47,4 @@ sleep 1
 
 echo 'Begin to run sanity test...'
 readonly CSI_SANITY_BIN='csi-sanity'
-"$CSI_SANITY_BIN" --ginkgo.v --csi.endpoint="$endpoint" --ginkgo.skip='should work|should fail when volume does not exist on the specified path|should be idempotent|pagination should detect volumes added between pages and accept tokens when the last volume from a page is deleted|should remove target path'
+"$CSI_SANITY_BIN" --ginkgo.v --csi.endpoint="$endpoint" --ginkgo.skip='should work|should fail when volume does not exist on the specified path|should be idempotent|pagination should detect volumes added between pages and accept tokens when the last volume from a page is deleted|should remove target path|should return appropriate capabilities'
```

Comment:
why `should return appropriate capabilities` case should be ignore here?

Reply:
this case will return error when modifyvolume capabilities 14 is not found in it

Reply:
It still has this error: https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_azuredisk-csi-driver/2344/pull-azuredisk-csi-driver-sanity/1813024904858046464

Reply:
[38;5;9m[FAILED][0m in [It] - /home/prow/go/src/github.com/kubernetes-csi/csi-test/pkg/sanity/controller.go:152 [38;5;243m@ 07/16/24 01:42:34.027[0m
[38;5;9m• [FAILED] [0.002 seconds][0m
[0mController Service [Controller Server] [38;5;243mControllerGetCapabilities [38;5;9m[1m[It] should return appropriate capabilities[0m
[38;5;243m/home/prow/go/src/github.com/kubernetes-csi/csi-test/pkg/sanity/controller.go:124[0m

  [38;5;9m[FAILED] Unknown capability: MODIFY_VOLUME
  [0m